sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFileFormat.scala

```diff
@@ -581,62 +586,6 @@ private[sql] object ParquetFileFormat extends Logging {
     }
   }
 
-  /** This closure sets various Parquet configurations at both driver side and executor side. */
-  private[parquet] def initializeLocalJobFunc(
```

Comment:
I discussed this with @yhuai offline, because the default value of split size (128M) could match the default value of group size (128M) in parquet, so we will not regression in most the cases. It's true that user may still have a different value in HadoopConf, we did not respect that anymore, I think it's fine.